SECURITY.md

```diff
@@ -53,6 +53,122 @@ Here is the security disclosure policy for Node.js
   the release process above to ensure that the disclosure is handled in a
   consistent manner.
 
+## The Node.js threat model
+
+Being able to cause a negative outcome in a way that requires control of the
+elements that are trusted by Node.js is not considered a vulnerability in
+Node.js.
+
+Being able to cause the following through control of the elements that Node.js
+does not trust is considered a vulnerability:
+
+* Disclosure or loss of integrity or confidentiality of data protected through
+  the correct use of Node.js APIs.
+* The unavailability of the runtime, including the unbounded degradation of its
+  performance.
+
+If Node.js loads configuration files or runs code by default (without a
+specific request from the user), and this is not documented, it is considered a
+vulnerability.
+Vulnerabilities related to this case may be fixed by a documentation update.
+
+**Node.js does NOT trust**
+
+1. The data from network connections that are created through the use of Node.js
```

Comment:
Is the data in question here the `protocol data` or the `business data` or both? I guess it is the protocol data, because that is the one which relates to the vulnerabilities?

Reply:
It's any data that Node.js acts on. Headers are a good example which does fall into protocal data, but if there was some way that crafting the business data could cause Node.js itself to do something unexpected/bad then that would also be covered.